frontend/src/components/Schedule/Schedule.tsx

```diff
@@ -82,7 +145,18 @@ const Schedule = () => {
     // console.log('dragged event:', event.title);
     // console.log('old resourceId:', event.resourceId);
     // console.log('new resourceId:', resourceId);
-    const nextEvents = events.map((old) => (old.id === event.id ? { ...old, resourceId } : old));
+    const updatedCalDriver = calDrivers.find(d => d.resourceId === resourceId);
+    const oldTitle = events.find(e => (e.id === event.id))?.title;
+    const unchanged = oldTitle?.substr(0, oldTitle?.indexOf("Driver"));
+    const title = `${unchanged}Driver: ${updatedCalDriver?.resourceTitle}`;
```

Comment:
can update event driver instead of updating title if you make the changes mentioned earlier